lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const _ = require('lodash');
+const traverse = require('traverse');
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Should contain a custom error message for multiple assertions.',
+      category: 'Possible Errors',
+      recommended: true
+    },
+    schema: []
+  },
+  create: context => {
+    return {
+      'ExpressionStatement': node => {
+
+        if (_.get(node, 'expression.callee.name') !== 'it') {
+          return;
+        }
+
+        const expectCallExpressions = [];
+        const expressionBodyNode = _.get(node, 'expression.arguments[1].body.body');
+
+        traverse(expressionBodyNode).forEach(function (thisNode) {
+          if (isChaiCall(thisNode) || isExpectCall(thisNode)) {
+            expectCallExpressions.push(thisNode);
+          }
+        });
+
+        if (expectCallExpressions.length > 1) {
+          if (_.find(expectCallExpressions, isExpectationMissingErrorMessage)) {
+            context.report({
+              node,
+              message: 'Should contain a custom error message due to multiple assertions existing in the same "it" block'
+            });
+          }
+        }
+      }
+    };
+  }
+};
+
```

Comment:
I'd prefer to keep it close to the code that uses it; we can move it out if we need to use it somewhere else.